roles/kubernetes/preinstall/tasks/0040-verify-settings.yml

```diff
@@ -42,11 +42,20 @@
 
 - name: Stop if unknown network plugin
   assert:
-    that: kube_network_plugin in ['calico', 'flannel', 'weave', 'cloud', 'cilium', 'cni', 'kube-ovn', 'kube-router', 'macvlan', 'custom_cni']
+    that: kube_network_plugin in ['calico', 'flannel', 'weave', 'cloud', 'cilium', 'cni', 'kube-ovn', 'kube-router', 'macvlan', 'custom_cni', 'none']
     msg: "{{ kube_network_plugin }} is not supported"
   when:
     - kube_network_plugin is defined
     - not ignore_assert_errors
+- name: Warn the user if they are still using `etcd_kubeadm_enabled`
```

Comment:
@ant31  Is this a typo?  How is the warning related to `etcd_kubeadm_enabled` ? It looks like this is only related to `kube_network_plugin`.

Reply:
This was fixed in #11901